src/haddock/libs/libplots.py

```diff
@@ -706,36 +832,129 @@ def clt_table_handler(clt_file, ss_file):
     fig :
         an instance of plotly.graph_objects.Figure
     """
+    # table of statistics
     dfcl = read_capri_table(clt_file)
     statistics_df = clean_capri_table(dfcl)
+
+    # table of structures
     structs_df = find_best_struct(ss_file, number_of_struct=10)
-    structs_df = _add_links(structs_df)
 
     # Order structs by best (lowest score) cluster on top
-    structs_df = structs_df.set_index('Cluster ID')
-    structs_df = structs_df.reindex(index=statistics_df['Cluster ID'])
+    structs_df = structs_df.set_index("Cluster ID")
+    structs_df = structs_df.reindex(index=statistics_df["Cluster ID"])
     structs_df = structs_df.reset_index()
 
-    fig = make_subplots(
-        rows=2,
-        cols=1,
-        specs=[[{"type": "table"}], [{"type": "table"}]],
-        vertical_spacing=0.1,
+    # Add download links and viewer
+    structs_links_df = _add_viewers(structs_df)
+
+    # Merge and convert dataframe to html table
+    df_merged = pd.merge(
+        statistics_df, structs_links_df, on=["Cluster ID", "Cluster Rank"]
         )
-    for i, df in enumerate([statistics_df, structs_df]):
-        table = go.Table(
-            columnwidth=100,
-            header=dict(values=list(df.columns), align="left"),
-            cells=dict(
-                values=df.transpose().values.tolist(), align="left", height=40
-                ),
-            )
-        fig.add_trace(table, row=i + 1, col=1)
-    fig.update_layout(title_text="Summary", height=600)
-    return fig
+
+    # The header of the table would be the cluster rank instead of id
+    df_merged = df_merged.set_index("Cluster Rank")
+    table = _pandas_df_to_html_table(df_merged.T, table_id="table")
+    return [table]
+
+
+def _css_styles_for_report():
```

Comment:
can you add a small explanation/docstring?

Reply:
added!